tslib/.eslintrc.js

```diff
@@ -34,8 +34,9 @@ module.exports = {
         'ts-nocheck': true,
         'ts-check': false,
         minimumDescriptionLength: 5
-      }
+      },
     ],
+    '@typescript-eslint/no-array-constructor': 'off',
```

Comment:
The constructor style is generally discouraged.  Why turn this off?

Reply:
fixed.  now using `[] as any`